src/System.Net.Http/src/netcore50/System/Net/cookie.cs

```diff
@@ -8,15 +8,15 @@
 using System.Text;
 using System.Threading;
 
-// The NETNative_SystemNetHttp #define is used in some source files to indicate we are compiling classes
+// The uap #define is used in some source files to indicate we are compiling classes
 // directly into the .NET Native System.Net.Http.dll implementation assembly in order to use internal class
 // methods. Internal methods are needed in order to map cookie response headers from the WinRT Windows.Web.Http APIs.
 // Windows.Web.Http is used underneath the System.Net.Http classes on .NET Native. Having other similarly
 // named classes would normally conflict with the public System.Net namespace classes that are also in the 
 // System.Private.Networking dll. So, we need to move the classes to a different namespace. Those classes are never
```

Comment:
Nit: System.Private.Networking dll does not exist anymore in case you would like to fix it.